script/add-payment-partials-to-marriage-abroad-services-data.rb

```diff
@@ -1,14 +1,26 @@
 countries = %w(
-  bulgaria
-  croatia
-  cyprus
-  guatemala
-  moldova
-  panama
-  venezuela
+  austria
+  brazil
+  czech-republic
+  denmark
+  ecuador
+  finland
+  germany
+  iceland
+  luxembourg
+  slovenia
+  sweden
 )
 
+# Exclude countries ignored in the outcome template
+countries = countries - %w(czech-republic cote-d-ivoire)
```

Comment:
I've updated the commit note to explain that the excluded countries are there to match the behaviour in the outcome template. See this same commit in PR #2511: https://github.com/alphagov/smart-answers/pull/2511/commits/1db82d9314a84ad1dce0a8c4e2e3c339e0e33bc7